test/services/DateTimeProviderServiceSpec.scala

```diff
@@ -14,21 +14,3 @@
  * limitations under the License.
  */
 
-package services
-
-import util.BaseSpec
-
-import java.time.LocalDate
-
-class DateTimeProviderServiceSpec extends BaseSpec {
```

Comment:
why you deleted these tests? I can see main class is still there anad not have a unit tests in place

Reply:
Changed